bundles/org.openhab.binding.miio/src/main/resources/database/viomi.vacuum.v18-miot.json

```diff
@@ -0,0 +1,1537 @@
+					"step": 1,
+					"pattern": "%.1f"
+				},
+				"refresh": true,
+				"actions": [
+					{
+						"command": "set_properties",
+						"parameterType": "STRING"
+					}
+				]
+			},
+			{
+				"property": "map-name",
+				"siid": 7,
+				"piid": 4,
+				"friendlyName": "Map - Map Name",
+				"channel": "map-name",
+				"type": "String",
+				"refresh": true,
+				"actions": [
+					{
+						"command": "set_properties",
+						"parameterType": "STRING"
+					}
+				]
+			},
+			{
+				"property": "cur-cleaning-path",
+				"siid": 7,
+				"piid": 10,
+				"friendlyName": "Map - Cur Cleaning Path",
+				"channel": "cur-cleaning-path",
+				"type": "String",
+				"stateDescription": {
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "map-list",
+				"siid": 7,
+				"piid": 11,
+				"friendlyName": "Map - Map List",
+				"channel": "map-list",
+				"type": "String",
+				"stateDescription": {
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "oper-result",
+				"siid": 7,
+				"piid": 13,
+				"friendlyName": "Map - Oper Result",
+				"channel": "oper-result",
+				"type": "String",
+				"stateDescription": {
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "auto-area-id",
+				"siid": 7,
+				"piid": 14,
+				"friendlyName": "Map - Auto Area Id",
+				"channel": "auto-area-id",
+				"type": "Number",
+				"stateDescription": {
+					"minimum": 0,
+					"maximum": 1410065407,
+					"step": 1,
+					"pattern": "%.1f",
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "target-voice",
+				"siid": 8,
+				"piid": 3,
+				"friendlyName": "Voice - Target Voice",
+				"channel": "target-voice",
+				"type": "String",
+				"refresh": true,
+				"actions": [
+					{
+						"command": "set_properties",
+						"parameterType": "STRING"
+					}
+				]
+			},
+			{
+				"property": "cur-voice",
+				"siid": 8,
+				"piid": 4,
+				"friendlyName": "Voice - Cur Voice",
+				"channel": "cur-voice",
+				"type": "String",
+				"stateDescription": {
+					"readOnly": true
+				},
+				"refresh": true,
+				"actions": []
+			},
+			{
+				"property": "download-status",
+				"siid": 8,
+				"piid": 5,
+				"friendlyName": "Voice - Download Status",
+				"channel": "download-status",
+				"type": "Number",
+				"stateDescription": {
+					"readOnly": true,
+					"options": [
+						{
+							"value": "0",
+							"label": "Free"
+						},
+						{
+							"value": "1",
+							"label": "Downloading"
+						}
+					]
+				},
+				"refresh": true,
+				"actions": [],
+				"readmeComment": "Value mapping [\"0\"\u003d\"Free\",\"1\"\u003d\"Downloading\"]"
+			},
+			{
+				"property": "download-progress",
+				"siid": 8,
+				"piid": 6,
+				"friendlyName": "Voice - Download Progress",
+				"channel": "download-progress",
+				"type": "Number",
```

Comment:
shouldn't this be a percentage?

Reply:
yes, that's very likely. The spec did not indicate a unit